docs/api.md

```diff
@@ -720,6 +721,18 @@ Shortcut for [`keyboard.down`](#keyboarddownkey-options) and [`keyboard.up`](#ke
     - `omitBackground` <[boolean]> Hides default white background and allows capturing screenshots with transparency. Defaults to `false`.
 - returns: <[Promise]<[Buffer]>> Promise which resolves to buffer with captured screenshot
 
+#### page.select(selector, value)
+- `selector` <[string]> A [selector] to query page for
+- `value` <[string]|[Array]<[string]>> A string, or array strings in case of `multiple`, of options to select. If the provided value cannot be matched to the `option` value, it tries to match against the `option` label instead.
```

Comment:
we usually do varargs to support both array and a single value. Check out [`ElementHandle.uploadFile`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#elementhandleuploadfilefilepaths)

Reply:
Regarding the matching against value / label, is this an innovation or it's already implemented in some other automation library?

Reply:
Regarding the matching against value / label, first time I saw it was on CasperJS.
GhostJS also borrows the idea.